device-types/HPE/Aruba-2930F-12G-PoEP-2SFPP.yaml

```diff
@@ -0,0 +1,78 @@
+---
+manufacturer: HPE
+model: Aruba 2930F-12G-PoE+-2SFP+
+slug: hpe-aruba-2930f-12g-poep-2sfpp
+part_number: JL693A
+u_height: 1
+is_full_depth: false
```

Comment:
Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

Reply:
updated